docs/trulens_eval/tracking/logging/where_to_log/log_in_snowflake.md

```diff
@@ -0,0 +1,62 @@
+# ❄️ Logging in Snowflake
+
+Snowflake’s fully managed [data warehouse](https://www.snowflake.com/en/data-cloud/workloads/data-warehouse/?utm_cta=website-homepage-workload-card-data-warehouse) provides automatic provisioning, availability, tuning, data protection and more—across clouds and regions—for an unlimited number of users and jobs.
+
+TruLens can write and read from a Snowflake database using a SQLAlchemy connection. This allows you to read, write, persist and share _TruLens_ logs in a _Snowflake_ database.
+
+Here is a _working_ guide to logging in _Snowflake_.
+
+## Install the [Snowflake SQLAlchemy toolkit](https://docs.snowflake.com/en/developer-guide/python-connector/sqlalchemy) with the Python Connector
+
+For now, we need to use a working branch of snowflake-sqlalchemy that supports sqlalchemy 2.0.
+
+!!! example "Install Snowflake-SQLAlchemy"
+
+    ```bash
+    # Clone the Snowflake github repo:
+    git clone git@github.com:snowflakedb/snowflake-sqlalchemy.git
+
+    # Check out the sqlalchemy branch:
+    git checkout SNOW-1058245-sqlalchemy-20-support
+
+    # Install hatch:
+    pip install hatch
+
+    # Build snowflake-sqlalchemy via hatch:
+    python -m hatch build --clean
+
+    # Install snowflake-sqlalchemy
+    pip install dist/*.whl
+    ```
+
+## Install TruLens
+
+For now, we need to use a working branch of TruLens that supports snowflake databases.
+
+!!! example "Install TruLens"
+
+    ```bash
+    pip uninstall trulens_eval -y # remove existing PyPI version
+    pip install git+https://github.com/truera/trulens@garett/snowflake-test#subdirectory=trulens_eval
```

Comment:
is this step still required?

Reply:
I'm just confused why Garett is getting roped into this haha.

Reply:
I copied from Garett's doc that Josh linked above and didn't realize his filepath is still here. I'll change it to the latest trulens release.